cpp/src/arrow/types/primitive-test.cc

```diff
@@ -214,10 +214,10 @@ void TestPrimitiveBuilder<PBoolean>::Check(
     ex_null_bitmap = nullptr;
   }
 
-  auto expected = std::make_shared<BooleanArray>(size, ex_data, ex_null_count,
-      ex_null_bitmap);
-  std::shared_ptr<BooleanArray> result = std::dynamic_pointer_cast<BooleanArray>(
-      builder->Finish());
+  auto expected =
+      std::make_shared<BooleanArray>(size, ex_data, ex_null_count, ex_null_bitmap);
```

Comment:
I turned off bin-packing for method arguments and parameters, as that seemed to be the style in the code base, if we turned those on we might get close to what you expected the 'ex_null_bitmap' would be aligned with 'size' instead of indented 4 spaces.  I prefer bin-packing but I don't have a strong preference.